src/engines/kotor/module.h

```diff
@@ -85,6 +87,8 @@ class Module : public KotOR::Object, public KotOR::ObjectContainer {
 	Area *getCurrentArea();
 	/** Return the currently playing PC. */
 	Creature *getPC();
+	/** Return the fade quad */
+	Graphics::Aurora::FadeQuad *getFadeQuad();
```

Comment:
Another missing period at the end of the doxygen comment there. :P

Also, you should make that method return a reference instead of a pointer.